packages/cli/src/codegen.ts

```diff
@@ -686,16 +700,61 @@ export function sortByName<T extends { artifact: { name: string } }>(artifacts:
   return artifacts.sort((a, b) => (a.artifact.name > b.artifact.name ? 1 : -1))
 }
 
+function genUtils(outDir: string) {
+  const utilsPath = path.join(outDir, 'utils.ts')
+  const source = `
+    import { ContractInstance, groupOfAddress } from '@alephium/web3'
+    import { DeployContractExecutionResult } from '@alephium/cli'
+
+    export function loadContractInstanceFromDeployments<T extends ContractInstance>(
```

Comment:
Because there are dependencies in `@alephium/cli` that only work in nodejs env, placing this function within `@alephium/cli` would lead to frontend issues. So I've moved this function to a separately generated file.